extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -35,20 +34,25 @@ public function index() {
 		wfProfileIn( __METHOD__ );
 
 		// Since this "Community_App" article won't be found, we need to manually say it's okay so that it's not a 404.
-		$this->response->setCode( self::RESPONSE_OK );
+		$this->response->setCode( static::RESPONSE_OK );
 
 		// Pull in the app-configuration (has data for all apps)
 		$appConfig = [];
 		$memcKey = wfMemcKey( static::$CACHE_KEY, static::$CACHE_KEY_VERSION );
```

Comment:
Data stored in memcache on this key can be shared between wikis. For better performance, let's use `wfSharedMemcKey`:
https://github.com/Wikia/app/blob/64f54f50dcc6b80b296b795c52d63ff476760a52/includes/wikia/GlobalFunctions.php#L691-L691